src/NServiceBus.AcceptanceTesting/Support/TaskExtensions.cs

```diff
@@ -8,6 +8,7 @@
 
     static class TaskExtensions
     {
+        //this method will not timeout a task if the debugger is attached.
```

Comment:
There were two reasons I decided against that:
1) it introduces the need to have comments for each parameter. Do we want to maintain that  going forward on this type of code?
2) I've seen push back on other PRs where XML comments were added to private/internal methods so I figured it would receive the same.

I'm more than willing to change if people agree that an XML comment is the right thing here.